prow/cluster/jobs/istio/cni/istio.cni.master.yaml

```diff
@@ -113,6 +113,54 @@ postsubmits:
           privileged: true
       nodeSelector:
         testing: test-pool
+  - annotations:
+      testgrid-alert-email: istio-oncall@googlegroups.com
+      testgrid-dashboards: istio-cni-postsubmits-master
+      testgrid-num-failures-to-alert: "1"
+    branches:
+    - ^master$
+    decorate: true
+    extra_refs:
+    - base_ref: master
+      org: istio
+      path_alias: istio.io/istio
+      repo: istio
+    name: cni-e2e-master
+    path_alias: istio.io/cni
+    spec:
+      containers:
+      - command:
+        - entrypoint
+        - make
+        - prow-e2e
+        image: gcr.io/istio-testing/istio-builder:v20190807-7d818206
+        name: ""
```

Comment:
Feel free to drop as its ignored anyway

Reply:
I think we would need to change the tag in the struct in kubernetes repo to be omit_if_empty for this